src/lib/components/chat/Chat.svelte

```diff
@@ -90,6 +90,8 @@
 	import Spinner from '../common/Spinner.svelte';
 	import ChatVerifier from './ChatVerifier.svelte';
 
+	const WEB_SEARCH_STATUS_KEY = 'web-search-status';
```

Comment:
```suggestion
	const WEB_SEARCH_STATUS_KEY = 'web-search-enabled-status';
```

I think we can use a more specific key

Reply:
okay.